src/components-v2/InfluenceVault/InfluenceVaultUtil.tsx

```diff
@@ -0,0 +1,112 @@
+import { InfluenceVaultConfig } from 'mobx/model/vaults/influence-vault-data';
+import mainnetDeploy from '../../config/deployments/mainnet.json';
+
+export const isInfluenceVault = (address: string): boolean => {
```

Comment:
not that it makes a difference, but in the future i'd prefer to explicitly export functions as functions

Reply:
sure